api/types/src/transaction.rs

```diff
@@ -325,6 +325,8 @@ pub struct TransactionInfo {
     pub accumulator_root_hash: HashValue,
     /// Final state of resources changed by the transaction
     pub changes: Vec<WriteSetChange>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub block_height: Option<U64>,
```

Comment:
Can you check this isn't being serialized in BCS anywhere?

I don't see this being used anywhere, what is it for?

Reply:
https://github.com/aptos-labs/aptos-core/pull/4100/files#diff-7d08501bf0be62c4d2f3bc04f00751430602f13b7072c37bd07088b7535e1097L247-R291

We'll add block height after converting the transactions.

Reply:
And i do think that that we will add this at some point? @banool aren't you adding it in some places?

Reply:
Nope, I had no intention of exposing block info anywhere outside of the blocks endpoints.